weblate/templates/base.html

```diff
@@ -233,9 +233,8 @@
             <b class="caret"></b>
           </a>
           <ul class="dropdown-menu">
-            <li class="dropdown-header" id="profile-name"><b>{{ user }}</b><br />@{{ user.username }}</li>
+            <li><a class="button" id="profile-name"> href "{% url 'user_page' user=user.username %}"><b>{{ user }}</b><br />@{{ user.username }}</a></li>
```

Comment:
This seems to mess up HTML markup and removes some used styling. But generally it makes sense, the Profile link is really confusing.